src/LearnositySdk/Request/DataApi.java

```diff
@@ -218,7 +218,6 @@ private JSONObject createResponseObject(Remote remote) throws Exception {
     	response.put("body", remote.getBody());
     	response.put("contentType",  remote.getContentType());
     	response.put("statusCode", remote.getStatusCode());
-    	response.put("error", remote.getError());
```

Comment:
@klauste we might want to look at why `getError` was removed, rather than removing this.

Reply:
Just checked it, I removed that so that the exceptions becomes visible to the client. So makes sense.